<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagementWidget</class>
 <widget class="QWidget" name="ManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>40</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="upperNavigator">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="topleftSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="correctState">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="muteSwitch">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="timerSpacer" native="true">
       <layout class="QVBoxLayout" name="timerAligner">
        <item>
         <widget class="QLabel" name="timeDisplay">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="progress">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lowerNavigator">
     <item>
      <widget class="QWidget" name="prevPageSpacer" native="true">
       <layout class="QVBoxLayout" name="prevPageAligner">
        <item>
         <widget class="QPushButton" name="prevPage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>130</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft JhengHei</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>← 上一頁</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="nextPageSpacer" native="true">
       <layout class="QVBoxLayout" name="nextPageAligner">
        <item>
         <widget class="QPushButton" name="nextPage">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>130</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft JhengHei</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>下一頁 →</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
